tests/integration/leases_test.go

```diff
@@ -0,0 +1,103 @@
+package integration
+
+import (
+	"encoding/json"
+	"github.com/Optum/dce-cli/client/operations"
+	"github.com/Optum/dce-cli/mocks"
+	"github.com/Optum/dce-cli/pkg/service"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/go-openapi/runtime"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	"testing"
+	"time"
+)
+
+func TestLeasesCreate(t *testing.T) {
+	cli := NewCLITest(t)
```

Comment:
Put together a quick happy-path test for `dce leases create`. Could/should be expanded in the future, to cover other use cases